tests/Rector/MethodCall/EloquentOrderByToLatestOrOldestRector/Fixture/fixture.php.inc

```diff
@@ -11,7 +11,9 @@ $query->orderBy('created_at');
 $query->orderBy('created_at', 'desc');
 $query->orderBy('submitted_at');
 $query->orderByDesc('submitted_at');
-$query->orderBy($column);
+$query->orderBy($allowed_variable_name);
+$query->orderBy($unallowed_variable_name);
+$query->orderBy('unallowed_column_name');
```

Comment:
Would probably be better to exist in their own file like [skip_if_not_eloquent_builder.php.inc](https://github.com/driftingly/rector-laravel/blob/main/tests/Rector/MethodCall/EloquentOrderByToLatestOrOldestRector/Fixture/skip_if_not_eloquent_builder.php.inc)

```php
// skip_if_column_not_allowed.php.inc
<?php

namespace RectorLaravel\Tests\Rector\Cast\DatabaseExpressionCastsToMethodCall\Fixture;

$query->orderBy($unallowed_variable_name);
$query->orderBy('unallowed_column_name');

?>
```

Reply:
I can do this, but I thought these would be unallowed not because they weren't an instance of Builder but because they weren't desired to be auto-renamed. Does it still make sense to move to a different file? I'm not familiar with best-practices in this regard.

Reply:
I think @peterfox is saying to create a file called `skip_if_column_not_allowed.php.inc` or `skip_if_column_not_configured.php.inc` similar to how he did with with the [EloquentOrderByToLatestOrOldestRector](https://github.com/driftingly/rector-laravel/tree/main/tests/Rector/MethodCall/EloquentOrderByToLatestOrOldestRector) fixtures.